src/test/java/uk/gov/hmcts/reform/coh/controller/AnswerControllerTest.java

```diff
@@ -371,10 +408,10 @@ public void testUpdateAnswersNonExistentQuestion() throws Exception {
     }
 
     @Test
-    public void testUpdateAnswersWhenAnswerSubmitted() throws Exception {
+    public void testUpdateAnswersWhenAnswerAlreadySubmitted() throws Exception {
         String json = JsonUtils.getJsonInput("answer/standard_answer");
```

Comment:
The PUT answer controller only accepts the request body with the answer_text & answer_state fields. I think it should be possible to update to answer state without modifying the answer text?

Reply:
Nah. A PUT is supposed to replace the whole entity